image.yaml

```diff
@@ -147,8 +148,6 @@ modules:
           - name: os.kieserver.s2i
           - name: os.kieserver.webapp
           - name: os.kieserver.chmod
-          - name: os-java-run
```

Comment:
Hey Rob, should we also remove these two modules from other images? Like here for example:
https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/rhpam71-dev/businesscentral/image.yaml#L131

Reply:
You'd have to test those changes.  Only the changes in the PR have been tested by QE.  That said, I believe all that module does now is redirect to jboss.container.java.jvm.bash, which includes the jvm gc and proxy related configuration scripts.  Those should also be included through one of the maven s2i, but you'd have to verify the build, etc.  All I did was refactor the core cct_modules.  I did not migrate any of the images, other than openjdk.  (This also helped prove that the changes were backwards compatible.)

Reply:
@spolti, @rcernich:

We can merge stuff into master for >= CE sprint-24 if it changes ONLY os.kieserver.* modules in that repo, which are for bpms/brms 6.x images.

For our rhdm/rhpam 7.x images, we only use the jboss-kie-* modules in that repo, but we should not be merging anything into there that won't work for our 7.1 release, which is tied to CE sprint-23. If the changes Rob made are only in CE sprint-24, we should not be changing our jboss-kie-* module scripts, or any rhdm or rhpam image.yaml files for the 7.1 release.

Reply:
Thanks @rcernich.